### ISSUE TITLE
Modify Install of Display Binary

### ISSUE THREAD

**Issue by aXb0t** [alexb7217/Pi-Rack-Pro_SKU_RM0004-Install#1]
Suggestion came from prior issue suggestion:

> Can I suggest that it is not necessary to compile the display binary on every boot? Just compile it once, move it to /usr/local/bin, delete the project files, and then change the rc.local entry to simply start the binary (/usr/local/bin/display &).

**Comment by alexb7217** [comment]
Yes this is a good idea. The original suggestion issue was deleted somehow. Thank you whoever suggested it.
